app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/messages'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/messages' do
+    @messages = Messages.messages
+    erb :'messages'
+  end
+
+  get '/add_message' do
```

Comment:
The RESTful (conventional) route for returning the new peep form would be `GET "/peeps/new"`

Reply:
or `GET "/messages/new"` if you prefer to call your class Message

Reply:
There are also some conventions around what you call your class methods here, though they are less consistent between languages.  In Ruby, you would expect to see

Message.all
Message.create

Possibly because these are the method names used in the dominant framework, Rails.